lib/tty/prompt/expander.rb

```diff
@@ -30,6 +30,7 @@ def initialize(prompt, options = {})
         @done         = false
         @status       = :collapsed
         @hint         = nil
+        @auto_hint = options.fetch(:auto_hint) { false }
```

Comment:
I'm not 100% sure if `auto_hint` is the best name for this, as this is not really 'automatic' as more showing hint on prompt startup. Also, I'm kind of reluctant to add more options...